babel.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          node: 'current',
```

Comment:
Why is this here?

Reply:
This is required for babel to work

Reply:
OK actually, now changed to not require a config file at all.  you have to explicitly say this in eslintrc